frontend/src/components/profile/my-tickets.tsx

```diff
@@ -0,0 +1,228 @@
+            id: data.id,
+            __typename: "AttendeeTicket",
+          }),
+          fields: {
+            tickets(existingTicketRefs, { DELETE }) {
+              return DELETE;
+            },
+          },
+        });
+      }
+    },
+  });
+
+  const updateTicketCallback = useCallback(
+    (id: string) => {
+      return () => {
+        const answers = tickets
+          .filter((item) => item.id == id)[0]
+          .item.questions.map((question) => {
+            let answer;
+            let option = null;
+            if (question.options.length > 0) {
+              option = question.options.filter(
+                (option) =>
+                  option.id == selectedProducts[id][0].answers[question.id],
+              )[0];
+              answer = option.name;
+            } else {
+              answer = selectedProducts[id][0].answers[question.id];
+            }
+
+            const data = {
+              answer: answer,
+              question: question.id,
+            };
+            if (option) {
+              data["options"] = [option.id];
+            }
+            return data;
+          });
+
+        updateTicket({
+          variables: {
+            conference: code,
+            input: {
+              id,
+              name: selectedProducts[id][0].attendeeName,
+              email: selectedProducts[id][0].attendeeEmail,
+              answers: answers,
+            },
+          },
+        });
+      };
+    },
+    [selectedProducts],
+  );
+
+  return (
+    <Box
+      sx={{
+        borderTop: "primary",
+      }}
+    >
+      <Box
+        sx={{
+          maxWidth: "container",
+          mx: "auto",
+          my: 5,
+          px: 3,
+        }}
+      >
+        <Heading mb={5} as="h2" sx={{ fontSize: 5 }}>
+          <FormattedMessage id="profile.myTickets" />
+        </Heading>
+        {tickets.length === 0 && (
+          <FormattedMessage
+            id="profile.myTickets.notickets"
+            values={{ linkTicket: <Link path="/tickets">here</Link> }}
+          />
+        )}
+        {tickets.length > 0 && (
+          <Table
+            headers={headers}
+            mobileHeaders={headers}
+            data={tickets}
+            keyGetter={(item) => item.id}
+            rowGetter={(item) => [
+              item.item.name,
+              item.name,
+              item.email,
+              <Box sx={{ m: -2 }}>
+                <Button
+                  onClick={() => setCurrentTicket({ id: item.id, show: true })}
+                  variant="small"
+                >
+                  <FormattedMessage id="profile.manageTicket" />
+                </Button>
+              </Box>,
+            ]}
+          />
+        )}
+
+        {updatedData?.updateAttendeeTicket.__typename === "AttendeeTicket" && (
+          <Alert variant="success">
+            <FormattedMessage
+              id={`profile.myTickets.update.AttendeeTicket.message`}
+            />
```

Comment:
```suggestion
            <FormattedMessage
              defaultMessage="English Message here"
              id="profile.myTickets.update.AttendeeTicket.message"
            />
```

I am trying to have the default english message in the code, what do you think?

Reply:
Mmmm I find it unnecessary
1. we will not notice if we are missing a translation because there is the default message
2. If we have to change the message we have to change in two places
3. Some messages are quite long...

Reply:
>If we have to change the message we have to change in two places

we already have to change it in two places

> we will not notice if we are missing a translation because there is the default message

not sure of this, we already missed this quite a lot of times

> Some messages are quite long...

not sure how much it matters

one pro side of having the message as `defaultMessage` is that you will be able to find the code much easier when searching the string instead of having to find the locale.ts file -> copying the key -> search again